src/ucp/wireup/wireup.c

```diff
@@ -360,6 +360,8 @@ ucp_wireup_init_lanes_by_request(ucp_worker_h worker, ucp_ep_h ep,
 {
     ucs_status_t status = ucp_wireup_init_lanes(ep, ep_init_flags, UINT64_MAX,
                                                 remote_address, addr_indices,
+                                                ucp_ep_has_cm_lane(ep) ?
+                                                ucp_ep_config(ep)->key.cm_idx :
```

Comment:
can't we always pass ucp_ep_config(ep)->key.cm_idx

Reply:
done

Reply:
unfortunately, returned back, since EP cfg_index could be NULL